integration/tests/cluster_dns/cluster_dns_test.go

```diff
@@ -0,0 +1,87 @@
+// +build integration
+
+package cluster_dns
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	. "github.com/onsi/gomega"
+	. "github.com/weaveworks/eksctl/integration/runner"
+	"github.com/weaveworks/eksctl/integration/utilities/kube"
+
+	"github.com/weaveworks/eksctl/integration/tests"
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	"github.com/weaveworks/eksctl/pkg/testutils"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var params *tests.Params
+
+func init() {
+	// Call testing.Init() prior to tests.NewParams(), as otherwise -test.* will not be recognised. See also: https://golang.org/doc/go1.13#testing
+	testing.Init()
+	params = tests.NewParams("cluster-dns")
+}
+
+func TestClusterDNS(t *testing.T) {
+	testutils.RegisterAndRun(t)
+}
+
+var _ = Describe("(Integration) [Cluster DNS test]", func() {
+
+	Context("Cluster with non-default ServiceIPv4CIDR", func() {
+		BeforeSuite(func() {
```

Comment:
are `XSuite` funcs supposed to live inside the actual test? 🤔  I know that _can_ but I think it is convention to put them outside the `Describe` at suite level

Reply:
They're typically defined at the top-level for global initialisation but we also have some tests defining it in a `Describe` block. I  chose to define it this way because it's also testing the `create cluster` functionality with a custom `ServiceIPv4CIDR` and having it in a `Describe` block lets you provide more context for the test. I was going to use `BeforeEach` but our integration tests have been using `BeforeSuite` for cluster creation.